README.md

```diff
@@ -72,3 +72,28 @@ DATE=20230427 rake make_package
 * Merge down to master
 * Release via github
 * run `rake release` from master
+
+# Recommandations for gem owners
+
+* Gem owners are encouraged to start releasing pre/rc packages on rubygems. It would avoid several issues
+    * Downloading from git is slow for large repositories
+    * It also avoids the issues where they could go into the .bundle/ruby/bundler folder and not .bundle/ruby/gems one (which we worked-around for openstudio-gems by repacking them manually...)
+
+## Need a native gem as a dependency and intend to use the CLI with --bundle
+
+Using a native gem as a dependency can cause conflicts in the CLI. This is because the CLI is statically built, but `bundle install` with your system ruby will build a shared extension.
+We have mostly worked-around this issue by ensuring that any native dependency in the Gemfile.lock that is satisfied by an embedded CLI native gem will be picked up.
+
+But this is fragile and prone to problems.
+
+Anything that is, or indirectly pulls, a native dependency that is already bundled with our CLI should be put in the `:test` group in the Gemfile (or added via `spec.add_development_dependency` which is the same result from the gempsec file). This is because by default `--bundle_without` is `[test]`.
+
+Or in a `:native` group, and you call the CLI with `--bundle_without "test native"`.
+
+That way we don't even run into such issues, the CLI version is used and that's it.
+
+If you definitely need a native gem that is NOT part of the CLI already, then either:
+
+* It should be part of the CLI anyways if you intend to use the CLI to run your thing... So request it, and we'll see if we can compile it statically (for MSVC especially)
+* Otherwise don't try to use the CLI with `--bundle` (via for eg `rake openstudio:test_with_openstudiowhich` uses a CLI call with `--bundle` underneath). 
```

Comment:
the '`' is in the wrong place on this line. should be: 
* Otherwise don't try to use the CLI with `--bundle` (via for eg `rake openstudio:test_with_openstudio` which uses a CLI call with `--bundle` underneath).